src/views/ConsentContinuationOut.vue

```diff
@@ -88,6 +88,15 @@
               </div>
             </section>
 
+            <section>
```

Comment:
I'll be removing this section before I merge. I just have it here so that you can play around with it and try it out in the temporary URL.

Reply:
Glad you have that in there for temporary testing (which looks good btw).

Reply:
Yes, remove when done because you don't do anything with the data and we don't want to leave in code that looks like it's doing something but it really isn't (decoy pattern).